_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -49,11 +49,15 @@ We have three major CMS projects:
 [Writer]
 [Genome tool]
 
-## Analytics
+## Data Pipeline
+
+We use a set of [Rake](https://github.com/ruby/rake) tasks run on [Jenkins](https://wiki.jenkins-ci.org/display/JENKINS/Build+Flow+Plugin) to move data from several MongoDB and PostgreSQL databases to [AWS RedShift](https://aws.amazon.com/redshift) via [S3](https://aws.amazon.com/s3/). We also store application usage data provided by [Segment Warehouses](https://segment.com/warehouses) as well as data from vendors such as [Salesforce](https://www.salesforce.com/) and [Sailthru](http://www.sailthru.com/).
 
-We have consolidated a lot of our analytics tooling into RedShift
+For production data services like recommendations we leverage [Apache Spark](http://spark.apache.org/) and [Cloudera Hadoop](https://www.cloudera.com/products/open-source/apache-hadoop.html).
```

Comment:
like -> such as ..., and I would definitely add a lot more examples of what we do